bot/cogs/core/paginator.py

```diff
@@ -11,13 +11,19 @@
 
 
 class PaginatorView(discord.ui.View):
-    def __init__(self, get_page, page_count, timeout):
+    def __init__(self, author_id, get_page, page_count, timeout):
```

Comment:
Can we have typehints here?

Reply:
Sure, I hadn't added them before because it uses practically the same arguments as paginate_embed.